pkg/storage/export/export/export.go

```diff
@@ -992,6 +996,40 @@ func (ctrl *VMExportController) getCertParams() (*CertParams, error) {
 	}, nil
 }
 
+func populateInitialVMExportStatus(vmExport *exportv1.VirtualMachineExport) {
+	if vmExport.Status == nil {
```

Comment:
I think its better to check if vmexport.Status != nil before calling this function, cause it will only happen once and its redundant to call this function every reconile.

Reply:
Yup

Reply:
Done